code/game/objects/items/stacks/medical.dm

```diff
@@ -122,8 +135,18 @@
 		//After the do_mob to ensure metabolites have had time to process at least one tick.
 		if(reagent && (C.reagents.get_reagent_amount(/datum/reagent/metabolite/medicine/styptic_powder) || C.reagents.get_reagent_amount(/datum/reagent/metabolite/medicine/silver_sulfadiazine)))
 			to_chat(user, "<span class='warning'>That stuff really hurt! You'll need to wait for the pain to go away before you can apply [src] to your wounds again, maybe someone else can help put it on for you.</span>")
+			C.balloon_alert(user, "You fail to apply [src] to yourself!")
 			return
 
+	if(stop_bleeding)
+		C.suppress_bloodloss(stop_bleeding)
+		if (C.is_bleeding())
+			C.balloon_alert(user, "You reduce [M == user ? "your" : M.p_their()] bleeding to [C.get_bleed_rate_string()]")
+		else
+			C.balloon_alert(user, "You stop [M == user ? "your" : M.p_their()]'s bleeding!")
```

Comment:
![typo1](https://github.com/BeeStation/BeeStation-Hornet/assets/98481852/ff690768-2910-40c3-bfa4-66da4423b249)
Typo. Should be without the ['s]

Reply:
Correct, thanks for spotting this